one-ui/src/main/java/com/marklogic/hub/oneui/services/DataHubService.java

```diff
@@ -29,6 +29,19 @@ public boolean install(HubDeployStatusListener listener) {
         return false;
     }
 
+    public boolean unInstall(HubDeployStatusListener listener) {
```

Comment:
Since the client for this method doesn't care about the return value, let's declare this as void. It appears the expectation is that if anything goes wrong, the listener will be told about it. So there's no value in returning true/false - the method is expected to have succeeded unless the listener is given an error.